var/spack/repos/builtin/packages/r/package.py

```diff
@@ -82,6 +82,8 @@ class R(Package):
     depends_on('pcre')
     depends_on('jdk')
 
+    patch('zlib.patch', when='@:3.3.1', level=0)
```

Comment:
I think the only difference would be that instead of:
```
*** configure.orig	2017-01-21 21:48:35.077000000 +0000
--- configure	2017-01-21 21:50:50.700000000 +0000\
```
it would look like:
```
--- a/configure
+++ b/configure
```
All of the other patches in Spack are level=1, so maybe it's just my OCD kicking in. It's up to you, I won't make you do it 😝